charts/invenio/templates/_helpers.tpl

```diff
@@ -334,7 +334,7 @@ INVENIO_SEARCH_HOSTS: {{ printf "[{'host': '%s'}]" (include "invenio.opensearch.
 INVENIO_SITE_HOSTNAME: '{{ include "invenio.hostname" $ }}'
 INVENIO_SITE_UI_URL: 'https://{{ include "invenio.hostname" $ }}'
 INVENIO_SITE_API_URL: 'https://{{ include "invenio.hostname" $ }}/api'
-INVENIO_DATACITE_ENABLED: "False"
+INVENIO_DATACITE_ENABLED: "{{ .Values.invenio.datacite.enabled }} "
```

Comment:
Before it was `"False"`, now it's `"false"`. Are both valid?

Reply:
They are not! The lower-case ones won't get interpreted as booleans, messing up the whole thing. 
I fixed that up already.